.github/workflows/codespell.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
This is probably fine.  It would be nice to include release branches, but should ignore renovate PR branches.  So keeping it simple like this is probably fine.